File: C/week 2/largerint.c
#include <stdio.h>

int main()
{
    int num1, num2, num3;
    printf ("We need Three numbers this time, first one: ");
    scanf ("%d", &num1);

    printf ("Now for the second number: ");
    scanf ("%d", &num2);

    printf ("And finally the third: ");
    scanf ("%d", &num3);

    printf ("Now lets find the largest number of the 3. \n\n");

    if (num1 > num2)
    {
        if (num1 > num3)
        {
            printf ("%d is the largest", num1);
        }
        else
        {
            printf ("%d is the largest", num3);
        }
        
    }
    else if (num2 > num3)
    {
        printf ("%d is the largest", num2);
    }
    else
    {
        printf ("%d is the largest", num3);
    }
    return 0;
    
}

File: Python/week 2/LargerInt.py
print("Lets see which of three numbers is the largest: ")
num1 = int(input("First number: "))
num2 = int(input("Second number: "))
num3 = int(input("Third number: "))

if num1 > num2 and num1 > num3:
    print(num1)
elif num2 > num1 and num2 > num3:
    print(num2)
else:
    print(num3)
